### ISSUE TITLE
Can I run SPMC alongside KODI to test it?

### ISSUE THREAD

**Issue by spmckiddy** [koying/SPMC#212]
Hi

Would like to move over to this due to having a Nvidia Sheild but wondered if I could run this alongside KODI 16 (from Googlestore) without any ramifications?

I.e. Does it use shared folders etc?

Thanks

**Comment by heymoe** [comment]
By default Kodi and SPMC do not share the same data directory so.

If you want to clone your current Kodi configuration you can copy the Kodi data directory structure into the SPMC data directory location. I personally did this when I moved from Kodi to SPMC on my Shield TV device.

**Comment by spmckiddy** [comment]
Thanks for that heymoe, I was planning on just installing it and sharing 1 folder just to test it out it all honesty! But good to know it won't affect any settings etc with my current Kodi install :)

**Comment by heymoe** [comment]
I should mention that if you are over-riding your Kodi data directory by using the xbmc_env.properties file, that SPMC will also follow that over-ride path so Kodi and SPMC would use that same data directory in that case.

http://kodi.wiki/view/HOW-TO:Change_data_location_for_Android

**Comment by koying** [comment]
Actually no. When using xbmc_env.properties, the root will be the same, but the data directory itself will be ".kodi" or ".spmc".
If you want the root to be different, use spmc_env.properties for SPMC.

**Comment by heymoe** [comment]
Ahhh.. Thanks @koying for the clarification.

**Comment by heymoe** [comment]
Looks like the Wiki page could be updated:

https://github.com/koying/SPMC/wiki/How-to-specify-the-location-of-SPMC-data%3F

Not sure if that is something I can do or not. I don't see an edit button or I'm overlooking it.

**Comment by spmckiddy** [comment]
So without getting too technical. Can I run SPMC (with different file shares) alongside Kodi without causing any issues to my Current Kodi setup?

**Comment by heymoe** [comment]
Yep, the sources.xml file would not be shared even if you used any of the data directory relocation stuff mentioned above.
